packages/loot-core/src/types/server-handlers.d.ts

```diff
@@ -31,8 +34,6 @@ export interface ServerHandlers {
 
   'transaction-add': (transaction) => Promise<EmptyObject>;
 
-  'transaction-add': (transaction) => Promise<EmptyObject>;
-
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏good catch